File: python/amazonbestsellers.py
import requests

task_params = {
    'target': 'amazon_bestsellers',
    'query': '2407760011',
    'domain': 'com',
    'locale':'en-us',
    'device_type':'desktop',
    'parse':'true'
}

username = 'SPusername'
password = 'SPpassword'

response = requests.post(
    'https://scraper-api.smartproxy.com/v2/scrape',
    json = task_params,
    auth = (username, password)
)

print(response.text)

File: nodejs/amazonbestsellers.js
const response = await fetch(
    'https://scraper-api.smartproxy.com/v2/scrape', {
        method: 'POST',
        body: {
            target: 'amazon_bestsellers',
            parse: true,
            query: '2407760011',
            domain: 'com',
            locale: 'en_us',
            device_type: 'desktop'
        },
        headers: {
            'Content-Type': 'application/json',
            'Authorization': 'Basic AUTH'
        },
    }
).catch(error => console.log(error));

console.log(response)
